national-resources/norway.md

```diff
@@ -0,0 +1,336 @@
+
+### Reports
+* [Weekly reports](https://www.fhi.no/en/publ/2020/weekly-reports-for-coronavirus-og-covid-19/) for COVID-19, influenza, and other respiratory tract infections
+* [National Strategy against Antibiotic Resistance 2015-2020](https://www.regjeringen.no/contentassets/5eaf66ac392143b3b2054aed90b85210/antibiotic-resistance-engelsk-lavopploslig-versjon-for-nett-10-09-15.pdf) (in PDF).
+* The renewed (2020) [Norwegian National Strategy against Antimicrobial Resistance](https://doi.org/10.3390%2Fantibiotics9090622)
+* [Overview of Norwegian Environment Agency reports on antimicrobial resistance in nature](https://www.miljodirektoratet.no/aktuelt/nyheter/2018/mars-2018/har-kartlagt-antibiotikaresistens-i-norsk-natur/) (in Norwegian)
+* The [NORM/NORM-VET report on antimicrobial use and resistance in Norway (2012)](https://unn.no/Documents/Kompetansetjenester,%20-sentre%20og%20fagr%C3%A5d/NORM%20-%20Norsk%20overv%C3%A5kingssystem%20for%20antibiotikaresistens%20hos%20mikrober/Rapporter/NORM%20NORM-VET%202021.pdf) (in PDF).
+
+## Dashboards and visualization platforms
+Several national registries, involved in tracking infectious disease cases, have their own dashboards for viewing data. All these platforms are available in Norwegian only:
+
+* Norwegian Immunisation Registry (SYSVAK) for [COVID-19](https://statistikk.fhi.no/sysvak/antall-vaksinerte?etter=diagnose&diagnose=COVID_19&fordeltPaa=maaned) and [influensa](https://statistikk.fhi.no/sysvak/antall-vaksinerte?etter=diagnose&diagnose=INFLUENSA&fordeltPaa=maaned) vaccinations.
+
+* [Norwegian Surveillance System for Communicable Diseases (MSIS)](https://statistikk.fhi.no/msis/sykdomshendelser?etter=diagnose&fordeltPaa=maaned) shows registered cases of infectious diseases in Norway by month, year, age, region, and more. The data is also available in [table format](https://msis.no/).
+
+* [Norwegian Surveillance System for Antimicrobial Drug Resistance (NORM) atlas](https://norm-atlas.no/) visualizes NORM database entries over time, divided by regions.
+
+## National data sources
+<!--- A section to list and provide context to national data sources. In the context of BY-COVID, a data source can be a repository which should include at least the metadata and ideally the data, that might not be directly available when considering sensitive data. Also, repositories should have the capacity to share this data and therefore have a governance model in place on how to do it. It can also include registries of data sources important for the field, with a direct link to the original data sources to be able to request access to the data. --->
+The FHI operates three national electronic reporting systems that are involved in surveillance of infectious diseases. Healthcare providers and laboratories use the following three systems to register cases, allowing health authorities to access data in real-time.
```

Comment:
I think it's a bit confusing that you say 3 projects here and then list four things (I understand that the fourth is an associated project and that is why it is listed here). I think that it'd be good to say e.g. 'three registries (then name them) and one associated (name it)' because then it'll be easy to understand what you're about to read

Reply:
I still find this a bit confusing, I think it could be made clearer with a slight change to the text to make it clear which are the three registries, just so it's clear why 4 are listed below

Reply:
Updated the section :)